build.sbt

```diff
@@ -3,7 +3,7 @@ import com.typesafe.sbt.SbtGit.{GitKeys => git}
 enablePlugins(GhpagesPlugin, SiteScaladocPlugin)
 
 name := "twitter4s"
-version := "7.2-SNAPSHOT"
+version := "8.0"
```

Comment:
Please revert this change, I'll make sure to take care of it when doing a release :)

Reply:
reverted